### DISCUSSION TITLE
fuzzy screen when using sanboot in ipxe

### DISCUSSION THREAD

**Discussion by smtadmin67** [ipxe/ipxe#1002] [General]
hi
i boot Sergei Strelec 2023.06.25 with ipxe, when the device boots up, the monitor according i attached images it's fuzzy, i tried with winpe10 and winpe11 in different devices and also changed resolution in 800*600 before but it's keep going on.
doyou have any ideas?!
thanks

![IMG_20230722_121445](https://github.com/ipxe/ipxe/assets/120118099/325fc7ca-f97f-446c-8767-800db26c7ed1)

![IMG_20230722_121737](https://github.com/ipxe/ipxe/assets/120118099/3d7b4e77-62c6-41b1-9ce5-22fc17b75460)

![IMG_20230722_122102](https://github.com/ipxe/ipxe/assets/120118099/04a96e97-bf5a-41c0-b927-952a78b6de38)

**Comment by NiKiZe** [comment]
That looks like a Windows issue, not sure why you think it has anything to do with iPXE?

**Reply by smtadmin67** [reply]
hi
as i say, i use sanboot for boot Sergei Strelec over iPXE, when i select it in boot menu this boot option will be appeared and when i select one of them the problem fuzzy boot screen will be happen.
i want an solution or advice to fix it
so can you help me?!

**Reply by NiKiZe** [reply]
And again, why would this have anything to do with iPXE?
Have you talked to the ones providing this unknown "Sergei Strelec" you don't think we are the developers?

If you had said that "this works fine when booting from USB" then maybe something could be seen as related, but currently, from my point of view, you are trying to get iPXE to help you with a software that don't have anything to do with iPXE.
